ui/src/app/components/webcomponents/dry-run-form.component.ts

```diff
@@ -405,7 +405,10 @@ export class DryRunFormComponent extends BaseComponent implements OnInit {
   }
 
   disableRunButton() {
-    return this.saving || this.savingConfig || this.emptyElements.length > 0 || !this.dryExecutionForm?.valid
+    if(this.isHybrid && this.dryExecutionForm.value.testDevices[0].platform==null)
```

Comment:
going 4 level nested would be risky, plz have a side effect validation @nixonakash01

Reply:
check latest commit please